### PR TITLE
ISSUE-131: Unrelated buttons on sidebar are hidden

### PR DESCRIPTION
**Pull request by arminio**
Fix for https://github.com/scalacenter/scastie/issues/131
Please review.

### CODE REVIEW

**Comment by MasseGuillaume** [comment]
@arminio Awesome! I will take a look in ~2h

File: client/src/main/scala/com.olegych.scastie.client/SideBar.scala [hunk 1]
```diff
@@ -68,6 +72,44 @@ object SideBar {
...
+
+        val console = li(onClick ==> toggleConsole,
+          title := s"$consoleLabel Console",
+          consoleSelected,
+          `class` := "button")(
+          iconic.terminal,
+          p("Console")
+        )
+
+        val buttonsRibbon: Map[View, Seq[TagMod]] = Map(
```
Comment (MasseGuillaume) [inline client/src/main/scala/com.olegych.scastie.client/SideBar.scala]: this should be a function `View => Seq[TagMod]`

File: server/src/main/resources/public/library-gray.svg [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
```
Comment (MasseGuillaume) [inline server/src/main/resources/public/library-gray.svg]: did you create this image? If not what's the license?

**Comment by MasseGuillaume** [comment]
@arminio Really good thanks. I only have two minor comments.

File: server/src/main/resources/public/library-gray.svg (hunk 2 above)
Comment (arminio) [inline server/src/main/resources/public/library-gray.svg]: I didn't create the Library svgs from scratch. I modified this:
http://www.flaticon.com/free-icon/library_201571#term=library&page;=1&position;=6
The original is free but it says the author needs to be credited. But I am not sure what the rules are for modifications like I have done.

File: client/src/main/scala/com.olegych.scastie.client/SideBar.scala (hunk 1 above)
Comment (arminio) [inline client/src/main/scala/com.olegych.scastie.client/SideBar.scala]: @MasseGuillaume done

**Comment by arminio** [comment]
@MasseGuillaume PR is amended after your comments 👍

File: server/src/main/resources/public/library-gray.svg (hunk 2 above)
Comment (MasseGuillaume) [inline server/src/main/resources/public/library-gray.svg]: can you add an xml comment for attribution?

File: client/src/main/scala/com.olegych.scastie.client/SideBar.scala [hunk 3]
```diff
@@ -68,6 +72,43 @@ object SideBar {
...
+            sharing
+          )
+          case View.Libraries => Seq(
+            LibraryButton(state, backend),
+            RunButton(state, backend)
+          )
+          case View.UserProfile => Seq(
+            LibraryButton(state, backend),
+            RunButton(state, backend),
+            li("User Profile (NY)")
```
Comment (MasseGuillaume) [inline client/src/main/scala/com.olegych.scastie.client/SideBar.scala]: NY ?

Comment (arminio) [inline client/src/main/scala/com.olegych.scastie.client/SideBar.scala]: @MasseGuillaume that was supposed to say NYI (not yet implemented)

**Comment by MasseGuillaume** [comment]
LGTM lets add the attribution latter

**Pull request merged**